packages/address/src.ts/index.ts

```diff
@@ -18,6 +18,18 @@ import {version} from "./_version";
 
 const logger = new Logger(version);
 
+export function asAccountString(acc :AccountLike): string {
```

Comment:
`AccountLike` can be:
```
0x0000000000000000000000000000000000000001
 0.0.1
 Account{shard:0, realm:0, num: 1}
 ```

From what I see, if someone sends `adsas` as string you are going to return that directly.
 I think that we should
- parse the string to account in order to verify that it is indeed account
- if it is not string, verify that it is `Account` object and only then return the shard, realm and num
- If it is not `account` nor string, throw unsupported error?